packages/@aws-cdk/aws-codebuild/lib/artifacts.ts

```diff
@@ -113,6 +113,14 @@ export interface S3ArtifactsProps extends ArtifactsProps {
    * @default true - files will be archived
    */
   readonly packageZip?: boolean;
+
+  /**
+   * If this is true, build output will not be encrypted.
+   * This is useful if the artifact to publish a static website or sharing content with others
+   *
+   * @default false - output will be encrypted
+   */
+  readonly encryptionDisabled?: boolean;
```

Comment:
We normally prefer to avoid double negatives, so rename this to “encrypted” and default to “true”

Reply:
I would also have preferred that, but didn't want it to differ from CloudFormation. Just out of curiosity, is clarity always a priority for the CDK, over existing definitions?